pkg/kubelet/kubelet.go

```diff
@@ -2798,6 +2800,10 @@ func (kl *Kubelet) canResizePod(pod *v1.Pod) (bool, v1.PodResizeStatus) {
 // the allocation decision and pod status.
 func (kl *Kubelet) handlePodResourcesResize(pod *v1.Pod, podStatus *kubecontainer.PodStatus) (*v1.Pod, error) {
 	allocatedPod, updated := kl.statusManager.UpdatePodFromAllocation(pod)
+	if goos == "windows" {
+		kl.statusManager.SetPodResizeStatus(pod.UID, v1.PodResizeStatusInfeasible)
+		return allocatedPod, nil
+	}
```

Comment:
Move this to after the `if !updated {` check. This function gets called on every SyncPod whether or not a resize is requested, so we only want to set the status if a resize is requested.

Reply:
Done.